environment_set_up/ManualConfig-HPC.md

```diff
@@ -0,0 +1,60 @@
+# Getting Started with On-Prem HPC
+
+This tutorial covers how to set up and use the Pangeo environment/stack
+on  USGS High Performance Computing (HPC) systems:
+
+1) Creating a custom hytest conda environment
+2) Configuring [Jupyter](https://jupyter.org/) server
+3) Launching a Jupyter server
+4) Connecting to your Jupyter server and the[Dask](https://dask.pydata.org/) 
+   dashboard from your personal computer
+
+This document assumes that you already have an access to
+Denali and Tallgrass and are comfortable using the command line.
+
+## Installing a Conda Software Environment
```

Comment:
Can we name this the same thing between the two files?

Reply:
I'm a little lost... what's "this" ? to name the same between which two files?

Reply:
Oh the section title...the text below says "This is the same manual process as describe in the 
[Quick-Start for HPC](./QuickStart-HPC.md)" but I don't see a section with that naming in the file it links to. Where do I find the instructions to install a conda software environment?

Reply:
I made a link directly to the header/anchor for the conda setup step.